gemma-web/src/main/java/ubic/gemma/web/services/rest/DatasetsWebService.java

```diff
@@ -325,6 +328,26 @@ public Response getDatasetExpression( // Params:
         }
     }
 
+    /**
+     * Retrieve raw expression data.
+     * @param datasetArg
+     * @return
+     */
+    @GZIP
+    @GET
+    @Path("/{dataset}/data/raw")
```

Comment:
It would be nice to have a coresponding `/{dataset}/data/processed` and permanently redirect `/{dataset}/data` there.